DirectProgramming/DPC++/Jupyter/oneapi-essentials-training/Makefile

```diff
@@ -1,199 +1,199 @@
 all: prog1 prog3 prog4 prog5 prog6 prog7 prog8 prog9 prog10 prog11 prog12 prog13 prog14 prog15 prog16 prog17 prog18 prog19 prog20 prog21 prog22 prog23 prog24 prog25 prog26 prog27 prog28 prog29 prog30 prog31 prog32 prog33 prog34 prog35 prog36 prog37 prog38 prog39 prog40 prog41 prog42 prog43 prog44 prog45 prog46 prog47 prog48 prog49 prog50 prog51 prog52 prog53 prog54 prog55 prog56 prog57 prog58 prog59 prog60 prog61 prog62 prog63 prog64 prog65 prog66
 
 prog1: 01_oneAPI_Intro/lab/simple.cpp
-	dpcpp -o 01_oneAPI_Intro/prog1 01_oneAPI_Intro/lab/simple.cpp
+	icpx -fsycl -o 01_oneAPI_Intro/prog1 01_oneAPI_Intro/lab/simple.cpp
```

Comment:
@KanclerzPiotr @praveenkk123 Maybe set a variable for the compiler and associated flags so you don't have to duplicate icpx -fsycl in the Makefile. Just a suggestion.

Reply:
I don't think this duplication is an issue here, with modern tools it's 5 seconds to find and replace all the occurrences.